src/Map.jsx

```diff
@@ -257,9 +264,11 @@ class Map extends Component {
           )
           : null,
         getRoutesLayer(geojson),
+        !(Object.keys(subroute).length === 0) && subroute.constructor === Object
+          ? getSubRoutesLayer(subroute)
+          : null,
```

Comment:
So there's a bug where:
1. I choose a stop on route A
2. I choose another stop on route A
3. I choose another stop on route B
-- the stop I selected on B gets selected (ie. green), the ones on A remain selected, then there's a crash.
Expected behaviour: stop on B gets selected, stops on A get deselected, no crash

crash:
```
Map.jsx:115 Uncaught TypeError: Cannot read property 'lat' of undefined
    at getCoordinateArray (Map.jsx:115)
    at Map.getRouteBetweenStops (Map.jsx:123)
    at Map.getStopInfo (Map.jsx:155)
    at Map.jsx:258
    at Object.onClick (Route.jsx:46)
    at draw-and-pick.js:214
    at Map.forEach (<anonymous>)
    at pickLayers (draw-and-pick.js:200)
    at LayerManager.pickLayer (layer-manager.js:279)
    at LayerManager._onClick (layer-manager.js:755)
    at Array.<anonymous> (event-manager.js:235)
    at Manager.emit (hammer.js:2519)
    at TapRecognizer.emit (hammer.js:2128)
    at TapRecognizer.tryEmit (hammer.js:1555)
    at TapRecognizer.recognize (hammer.js:1602)
    at Manager.recognize (hammer.js:2368)
    at PointerEventInput.inputHandler [as callback] (hammer.js:517)
    at PointerEventInput.PEhandler (hammer.js:909)
    at PointerEventInput.handler (hammer-overrides.js:55)
    at Input.domHandler (hammer.js:430)
```